cmd/explaintest/r/explain_easy.result

```diff
@@ -450,7 +450,7 @@ drop table if exists t;
 create table t(a bigint primary key);
 explain select * from t where a = 1 and a = 2;
 id	count	task	operator info
-TableDual_5	0.00	root	rows:0
+TableDual_5	8000.00	root	rows:0
```

Comment:
It's stange that the row count changed

Reply:
I  don't know why row count was 8000.00, but it just happened when I run `./run-tests -r all ../../bin/tidb-server` .

Reply:
done